maintainers/maintainer-list.nix

```diff
@@ -18338,6 +18338,15 @@
       fingerprint = "D2A2 F0A1 E7A8 5E6F B711  DEE5 63A4 4817 A52E AB7B";
     }];
   };
+  theaninova = {
```

Comment:
`a` should sorted be after `-`

Reply:
Hm, that's how I had it at first but the "check maintainer list is sorted" check yelled at me that I should put it here.

Reply:
hmm ok, let me look into it, at least my editor and sorted - before a :-)

Reply:
ok, I've learned something new ! maintainers/scripts/check-maintainers-sorted.nix removes `-` and `_` so `thean` comes before `the-ar`

Reply:
Huh, so I suppose that sorting is correct then?

Reply:
yes, sorting is correct